bitcraze_crazyflie_2/cf2.xml

```diff
@@ -0,0 +1,119 @@
+<mujoco model="cf2">
+  <option integrator="RK4"/>
+  <compiler inertiafromgeom="false" coordinate="local" meshdir="cf2"/>
```

Comment:
coordinate="local" is the default (in fact global is now deprecated) so please remove.